assets/base.css

```diff
@@ -1101,13 +1101,13 @@ summary::-webkit-details-marker {
   }
 }
 
-.grid--full-width {
+.grid--1-col-desktop {
   flex: 0 0 100%;
   max-width: 100%;
 }
 
 @media screen and (min-width: 990px) {
-  .grid--full-width .grid__item {
+  .grid--1-col-desktop .grid__item {
     width: 100%;
     max-width: 100%;
   }
```

Comment:
I'm not a huge fan of a desktop specific class affecting non-desktop breakpoints. Particularly if the class is actually relied upon by non-desktop breakpoints, which I think this is. Can we remove the first one here and just apply the same needed styles to `.grid--1-col-tablet-down`?

Reply:
That's ideal, as long as mobile/tablet weren't counting on the `flex: 0 0 100%;` from this class

Reply:
This was always only for desktop. It was accidentally kept outside of the media query but it only affects desktop